tests/bsim/bluetooth/mesh/prj.conf

```diff
@@ -74,4 +74,5 @@ CONFIG_BT_TESTING=y
 # Needed for RPR tests due to huge amount of retransmitted messages
 CONFIG_BT_MESH_MSG_CACHE_SIZE=64
 
-CONFIG_MBEDTLS_PSA_KEY_SLOT_COUNT=64
+# Enable tinycrypt as a crypto backend
```

Comment:
Is it necessary to remove "CONFIG_MBEDTLS_PSA_KEY_SLOT_COUNT"?

Reply:
It is set in psa overlay. No necessity to keep the option in the file which it doesn't belong.